doc/source/deployment/setup-deployer.rst

```diff
@@ -59,22 +59,15 @@ Deployer. Set up your workspace with the following steps:
 Installing the SUSE Containerized OpenStack software
 ----------------------------------------------------
 
-There are several ways to install the SUSE Containerized OpenStack software.
+There are two ways to install the SUSE Containerized OpenStack software.
 
 1. (Recommended) Install with an ISO image including required dependencies:
 
-   a. Download ``openSUSE-Addon-socok8s-x86_64-Media.iso`` from
-      https://download.opensuse.org/repositories/Cloud:/socok8s:/master/images/iso/
-   b. sudo zypper addrepo --refresh <PATH_TO_ISO_IMAGE> socok8s-iso
+   a. wget https://download.opensuse.org/repositories/Cloud:/socok8s/images/iso/openSUSE-Addon-socok8s-x86_64-Media.iso
+   b. sudo zypper addrepo --refresh iso:///?iso=/home/stack/openSUSE-Addon-socok8s-x86_64-Media.iso socok8s-iso
    c. sudo zypper install socok8s (installs to /usr/share/socok8s)
 
-2. Install from the openSUSE repository including required dependencies:
```

Comment:
why taking out the rpm package install approach from the master branch?

Reply:
Should this alternative be available in master branch? It was removed previously and is not available in stable/1.0.